administrator/language/en-GB/com_guidedtours.ini

```
@@ -0,0 +1,98 @@
+; Joomla! Project
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+COM_GUIDEDTOURS="Guided Tours"
+COM_GUIDEDTOURS_ACCESS_GUEST="Guest"
+COM_GUIDEDTOURS_ACCESS_PUBLIC="Public"
+COM_GUIDEDTOURS_ACCESS_REGISTERED="Registered"
+COM_GUIDEDTOURS_ACCESS_SPECIAL="Special"
+COM_GUIDEDTOURS_ACCESS_SUPER_USERS="Super users"
+COM_GUIDEDTOURS_BASIC_STEP="Basic Step"
+COM_GUIDEDTOURS_DESCRIPTION="Description"
+COM_GUIDEDTOURS_DESCRIPTION_ASC="Description ascending" 
+COM_GUIDEDTOURS_DESCRIPTION_DESC="Description descending"
+COM_GUIDEDTOURS_EDIT_TOUR="Edit Tour"
+COM_GUIDEDTOURS_ERROR_TOUR_NOT_FOUND="Tour not found!"
+COM_GUIDEDTOURS_EXTENSIONS_DESC="Will show the tour for the selected extensions in priority."
+COM_GUIDEDTOURS_EXTENSIONS_LABEL="Component Selector"
+COM_GUIDEDTOURS_FIELD_NOTE_LABEL="Note"
+COM_GUIDEDTOURS_FIELD_VALUE_BOTTOM="Bottom"
+COM_GUIDEDTOURS_FIELD_VALUE_CENTER="Center"
+COM_GUIDEDTOURS_FIELD_VALUE_INTERACTIVE_STEP="Interactive"
+COM_GUIDEDTOURS_FIELD_VALUE_LEFT="Left"
+COM_GUIDEDTOURS_FIELD_VALUE_NEXT_STEP="Next"
+COM_GUIDEDTOURS_FIELD_VALUE_REDIRECT="Redirect"
+COM_GUIDEDTOURS_FIELD_VALUE_RIGHT="Right"
+COM_GUIDEDTOURS_FIELD_VALUE_TOP="Top"
+COM_GUIDEDTOURS_FILTER_SEARCH_DESC="Search in id, title and description. Prefix with ID: to search for tour ID or DESCRIPTION: to search only description text."
+COM_GUIDEDTOURS_GUIDEDTOUR_STEPS_PAGE_TITLE="Guided Tour - List of Steps"
+COM_GUIDEDTOURS_GUIDEDTOURS_LABEL_DESCRIPTION="Description"
+COM_GUIDEDTOURS_GUIDEDTOURS_PAGE_TITLE="Guided Tours - List of Tours"
+COM_GUIDEDTOURS_GUIDEDTOURS_TABLE_CAPTION="List of Tours"
+COM_GUIDEDTOURS_INTERACTIVE_STEP_DESC="Click on the type of interactive step which you want to select"
+COM_GUIDEDTOURS_LANGUAGE="Language"
+COM_GUIDEDTOURS_LANGUAGE_ASC="Language ascending"
+COM_GUIDEDTOURS_LANGUAGE_DESC="Language descending"
+COM_GUIDEDTOURS_MANAGER_STEP_EDIT="Edit step"
+COM_GUIDEDTOURS_MANAGER_STEP_NEW="New step"
+COM_GUIDEDTOURS_MANAGER_TOUR_EDIT="Edit tour"
+COM_GUIDEDTOURS_MANAGER_TOUR_NEW="New tour"
```

```suggestion
COM_GUIDEDTOURS_MANAGER_STEP_EDIT="Edit Step"
COM_GUIDEDTOURS_MANAGER_STEP_NEW="New Step"
COM_GUIDEDTOURS_MANAGER_TOUR_EDIT="Edit Tour"
COM_GUIDEDTOURS_MANAGER_TOUR_NEW="New Tour"
```

should be capitalised accorfinh to the styleguide